schema/changelog-3.10.xml

```diff
@@ -48,5 +48,21 @@
       <column name="network" type="VARCHAR(4000)" />
     </addColumn>
 
+    <addColumn tableName="users">
+      <column name="userlimit" type="INT" defaultValueNumeric="0" />
+    </addColumn>
+
+    <createTable tableName="user_user">
+      <column name="userid" type="INT">
+        <constraints nullable="false" />
+      </column>
+      <column name="otheruserid" type="INT">
```

Comment:
Agree, I can't think of too. Only may be `slaveUserId`, If it is not better then I'll use `managedUserId`